internal/querynodev2/segments/mock_data.go

```diff
@@ -330,7 +335,7 @@ func GenTestIndexInfoList(collectionID int64, schema *schemapb.CollectionSchema)
 			TypeParams: field.GetTypeParams(),
 		}
 		switch field.GetDataType() {
-		case schemapb.DataType_FloatVector, schemapb.DataType_Float16Vector:
+		case schemapb.DataType_FloatVector, schemapb.DataType_Float16Vector, schemapb.DataType_BFloat16Vector:
```

Comment:
also the data type should be "Fp16Vector" and "Bf16Vector"

Reply:
fp16 and fp16 is nickname of knowhere，float16 and bfloat16 is more common.